### PR TITLE
[ReadMe] updated readMe file for Persistent-QuoteAnalitics

### PR DESCRIPTION
**Pull request by vlmed**
### Description (*)
Update README.md file for modules:

- Magento_Persistent
- Magento_ProductAlert
- Magento_ProductVideo
- Magento_Quote
- Magento_QuoteAnalitics

### Fixed Issues (if relevant)

1. magento/devdocs 4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Persistent/README.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
-Magento\Persistent module enables set customer a long-term cookie containing internal id (random hash - to exclude brute
+# Magento_Persistent module
+
+This module enables set customer a long-term cookie containing internal id (random hash - to exclude brute
```
Comment (bgorski) [inline app/code/Magento/Persistent/README.md]: Please rephrase this to "this module enables setting ... for a customer" or something similar. "This module enables set customer ..." doesn't look valid.

File: app/code/Magento/Persistent/README.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
 num items in shopping cart). For registered customer this info is synced to persistent session if choose "Remember me"
 checkbox during first login.
+
+## Installation
+
+Before installing this module, note that the Magento_Persistent is dependent on the following modules:
+- `Magento_Checkout`
+- `Magento_PageCache`
+
+The Magento_Persistent module creates the `persistent_session` table in the database.
```
Comment (bgorski) [inline app/code/Magento/Persistent/README.md:15]: Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.

File: app/code/Magento/ProductAlert/README.md [hunk 3]
```diff
@@ -1 +1,43 @@
-The Magento_ProductAlert module enables product alerts, which allow customers to sign up for emails about product price or stock status change.
+# Magento_ProductAlert module
+
+This module enables product alerts, which allow customers to sign up for emails about product price or stock status change.
+
+## Installation
```
Comment (bgorski) [inline app/code/Magento/ProductAlert/README.md:5]: Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.
Additionally, please add information about the recurring script in the module and if its modification need to be manually reverted upon uninstallation.

File: app/code/Magento/ProductVideo/README.md [hunk 4]
```diff
@@ -1 +1,44 @@
...
+
+This module implements functionality related with linking video files from external resources to product.
+
+## Installation
+
+Before installing this module, note that the Magento_ProductAlert is dependent on the following modules:
+- `Magento_Catalog`
+- `Magento_Backend`
+
+The Magento_ProductVideo module creates the `catalog_product_entity_media_gallery_value_video` table in the database.
```
Comment (bgorski) [inline app/code/Magento/ProductVideo/README.md:11]: Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.

File: app/code/Magento/Quote/README.md [hunk 5]
```diff
@@ -1,11 +1,310 @@
...
+This module provides customer cart management functionality.
 
-# Deployment
-## System requirements
+## Installation
 
-The Magento_Quote module does not have any specific system requirements.
+The Magento_Quote module is one of the base Magento 2 modules. You cannot disable or uninstall this module.
+
+The following modules depend on this module:
```
Comment (bgorski) [inline app/code/Magento/Quote/README.md]: I'm not sure if listing dependent modules here is a good idea. The README.md file obviously requires more maintenance now. However, it's relatively easy to maintain it when changes are made inside the module itself, but can be challenging in case of this list as it would require maintaining it when other modules are modified or added. Sometimes that would require adding commits/PRs to more than one project (e.g. if a new EE module depends on this one).

File: app/code/Magento/ProductAlert/README.md [hunk 6]
```diff
@@ -1 +1,43 @@
...
+
+Before installing this module, note that the Magento_ProductAlert is dependent on the following modules:
+- `Magento_Catalog`
+- `Magento_Customer`
+
+The Magento_ProductAlert module creates the following table in the database:
```
Comment (bgorski) [inline app/code/Magento/ProductAlert/README.md]: A typo - "table" should be "tables"

File: app/code/Magento/Quote/README.md [hunk 7]
```diff
@@ -1,11 +1,310 @@
...
+- `Magento_GroupedProduct`
+- `Magento_OfflineShipping`
+- `Magento_Paypal`
+- `Magento_PaypalGraphQl`
+- `Magento_QuoteAnalytics`
+- `Magento_Reports`
+- `Magento_SalesRule`
+- `Magento_Weee`
+
+The Magento_Quote module creates the following table in the database:
```
Comment (bgorski) [inline app/code/Magento/Quote/README.md:9]: Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.

Also - a typo - "table" should be in the plural form. 

Additionally, please cover all the data patches here and mention if modifications introduced in them can have to be manually removed/reverted after the module uninstallation.

File: app/code/Magento/Quote/README.md [hunk 8]
```diff
@@ -1,11 +1,310 @@
...
+    - quote items totals data
+
+#### General
+
+- `\Magento\Quote\Api\ChangeQuoteControlInterface`
+    - checks if user is allowed to change the quote
+    
+#### Guest
+
+- `\Magento\Quote\Api\GuestBillingAddressManagementInterfacee`
```
Comment (bgorski) [inline app/code/Magento/Quote/README.md]: a typo - Interfacee instead of Interface (not that it matters that much, but still)

File: app/code/Magento/QuoteAnalytics/README.md [hunk 9]
```diff
@@ -1,3 +1,17 @@
-# Magento_QuoteAnalytics
+# Magento_QuoteAnalytics module
 
-The Magento_QuoteAnalytics module configures data definitions for a data collection related to the Quote module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.4/advanced-reporting/modules.html).
+This module configures data definitions for a data collection related to the Quote module entities to be used in [Advanced Reporting](https://devdocs.magento.com/guides/v2.3/advanced-reporting/modules.html).
+
+## Installation
```
Comment (bgorski) [inline app/code/Magento/QuoteAnalytics/README.md:5]: Please add a mention that this module doesn't introduce any db modifications or new data (so everyone reading the readme knows that this is what happens - a lack of any mention might be considered just an omission, not as info that nothing gets added).

**Review by bgorski — CHANGES_REQUESTED** [review]

**Comment by bgorski** [comment]
@vlmed good job adding all those readme files! I had a few comments, but the overall effect looks great!

**Comment by bgorski** [comment]
@magento run all tests

**Comment by bgorski** [comment]
@magento run WebAPI Tests, Unit Tests, Functional Tests B2B

File: app/code/Magento/Persistent/README.md [hunk 10]
```diff
@@ -1,5 +1,63 @@
...
+
+#### Observer
+
+- `persistent_session_expired` event in the `\Magento\Persistent\Observer\CheckExpirePersistentQuoteObserver::execute` method
+
+For information about an event in Magento 2, see [Events and observers](http://devdocs.magento.com/guides/v2.4/extension-dev-guide/events-and-observers.html#events).
+
+### Layouts
+
+This module introduces the following layouts in the `view/frontend/layout` directory:
```
Comment (bgorski) [inline app/code/Magento/Persistent/README.md]: Actually it doesn't introduce those - it's only a "client" of layout handles introduced by Magneto_Customer - please remove them from here.
I initially haven't noticed that this comment should be added here, but better late than never.

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Comment by engcom-Foxtrot** [comment]
@bgorski updated, please review.

File: app/code/Magento/Quote/README.md (hunk 7 above)
Comment (engcom-Foxtrot) [reply app/code/Magento/Quote/README.md:9]: I don't think we should mention any schema / data reverting as this module should not be disabled or removed.

**Review by bgorski — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @bgorski, thank you for the review.
[ENGCOM-9026](https://jira.corp.magento.com/browse/ENGCOM-9026) has been created to process this Pull Request
:eight_spoked_asterisk: @bgorski, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Foxtrot** [comment]
False-positive Functional Tests EE failure. PR changes cannot affect MFTF tests as it's only modifying readme files.
